### PR TITLE
flex-wrap as a prop

### PR DESCRIPTION
**Pull request by guibarscevicius**
Add flex-wrap: nowrap as a prop (defaults to 'wrap')

### CODE REVIEW

File: components/Grid.vue [hunk 1]
```diff
@@ -18,6 +18,7 @@ export default {
     vertical: String,
     flat: String,
     pair: String,
+    nowrap: String,
```
Comment (bartkozal) [inline components/Grid.vue:21]: You could specify the default value here:

```js
wrap: {
  type: String,
  default: 'wrap'
}
```

File: components/Grid.vue [hunk 2]
```diff
@@ -54,7 +55,8 @@ export default {
         marginLeft: this.horizontalMargin,
         justifyContent: this.gridHorizontal,
         alignItems: this.gridVertical,
-        flexDirection: this.gridDirection
+        flexDirection: this.gridDirection,
+        flexWrap: isUndefined(this.nowrap) ? 'wrap' : 'nowrap'
```
Comment (bartkozal) [inline components/Grid.vue:59]: So with the default value initially set this one can be simplified to:

```js
flexWrap: this.wrap
```

**Review by bartkozal — CHANGES_REQUESTED** [review]
Hey, thank you so much for your contribution! 🙇 

I left two comments to make the code a bit cleaner. Can I also ask you to update tests and mention the new prop in the documentation? You will find necessary scripts to work on them in package.json.

**Comment by guibarscevicius** [comment]
Sorry, dont know what happened with the branches. Reopening the PR with changes.

**Pull request closed without merging**